app/test/testMetadata.js

```diff
@@ -6,29 +6,31 @@ const {app, errorMessages} = require('../service');
 chai.use(chaiHttp);
 chai.should();
 
-const exampleMetadata = {
-  url: 'http://www.example.com/good',
-  original_url: 'http://www.example.com/good',
-  title: 'An Example Page',
-  description: 'An example description',
-  favicon_url: 'http://www.example.com/rich-icon.png',
-  images: [
-    {
-      'url': 'http://www.example.com/preview.png',
-      'entropy': 1,
-      'height': 500,
-      'width': 500,
-    }
-  ],
-};
-
 const goodExampleUrl = 'http://www.example.com/good';
 const badExampleUrl = 'http://www.example.com/bad';
 
 function clone(original) {
   return JSON.parse(JSON.stringify(original));
 }
 
+function getExampleMetadata() {
+  return clone({
```

Comment:
Sure, I'll remove `clone()` here, but we're also using that function on L139 and L210 below, so we still need it around [I think].